### PR TITLE
Requirements for assignment 2

### LINKED ISSUE #107: Create the requirements document for the new features
The new features include improving physics, improving collisions, warping and jumping on bubbles.

### PR DESCRIPTION
**Pull request by LilideBree**
Fixes #107.

### CODE REVIEW

**Comment by toinehartman** [comment]
:+1: :shipit:

File: Documentation/Assignment 2/tex/assign_2_functional.tex [hunk 1]
```diff
@@ -0,0 +1,46 @@
+\chapter{Functional Requirements}
+
+\section{Introduction}
+In this document the requirements for assignment 2 will be stated. There will be improvements to the code, as well as new functionalities. \noindent This will be done using the \fnurl{MoSCoW Model}{https://en.wikipedia.org/wiki/MoSCoW_method}.
+
+\section{Must Have}
+\begin{itemize}
+\itemsep0em
+\item The player must be able to fall through the floor.
```
Comment (Verzijl) [inline Documentation/Assignment 2/tex/assign_2_functional.tex]: You don't want the player to fall through the floor, you want it to fall through a hole in the floor.

File: Documentation/Assignment 2/tex/assign_2_functional.tex [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+\item The player must reappear after falling through the floor at the corresponding square in the ceiling.
```
Comment (Verzijl) [inline Documentation/Assignment 2/tex/assign_2_functional.tex]: Again "hole in floor" and here it is corresponding "hole in ceiling"

File: Documentation/Assignment 2/tex/assign_2_functional.tex [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+\item The monsters must be able to fall though the floor.
```
Comment (Verzijl) [inline Documentation/Assignment 2/tex/assign_2_functional.tex]: "Hole in floor"

File: Documentation/Assignment 2/tex/assign_2_functional.tex [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+\item The monsters must reappear after falling through the floor at the corresponding square in the ceiling.
```
Comment (Verzijl) [inline Documentation/Assignment 2/tex/assign_2_functional.tex]: "Hole in floor" and "corresponding hole in ceiling"

File: Documentation/Assignment 2/tex/assign_2_functional.tex [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+\item The monsters must be able to jump.
```
Comment (Verzijl) [inline Documentation/Assignment 2/tex/assign_2_functional.tex]: This line can be deleted. The same thing with extra's is said in the line below.

File: Documentation/Assignment 2/tex/assign_2_functional.tex [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+\item The monsters must be able to fall in a more natural way, so an increase in speed when the monster falls.
+\item The bubbles must be able to float to the top of the screen.
+\item The bubbles must disappear after a period of time.
+\item The player must be able to jump on a bubble, and use it like a lift.
+\end{itemize}
+
+\section{Should Have}
+\begin{itemize}
+\itemsep0em
+\item The bubbles should be able to float through the ceilings, but stop at the top ceiling.
```
Comment (Verzijl) [inline Documentation/Assignment 2/tex/assign_2_functional.tex]: "The bubbles should be able to float through the platforms, but should stop at the ceiling."

**Comment by jimhommes** [comment]
If you fix Matthijs his comments, :shipit:

**Pull request merged**